virtual-server/service-worker.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-unused-vars: 0 */
+/* global importScripts, ServiceWorkerWare */
+importScripts('./lib/ServiceWorkerWare.js');
+
+// These are the default quotations. `makeDefaults()` simply add the id and the
+// sticky flag to make them non removables.
+var quotations = makeDefaults([
+  {
+    text: 'Humanity is smart. Sometime in the technology world we think we are smarter, ' +
+      'but we are not smarter than you.',
+    author: 'Mitchell Baker'
+  },
+  {
+    text: 'A computer would deserve to be called intelligent if it could deceive a human ' +
+      'into believing that it was human.',
+    author: 'Alan Turing'
+  },
+  {
+    text: 'If you optimize everything, you will always be unhappy.',
+    author: 'Donald Knuth'
+  },
+  {
+    text: 'If you don\'t fail at least 90 percent of the time, you\'re not aiming high enough',
+    author: 'Alan Kay'
+  },
+  {
+    text: 'Colorless green ideas sleep furiously.',
+    author: 'Noam Chomsky'
+  }
+]);
+
+// Adds id and the sticky flag to a list of quotes.
+function makeDefaults(quotationList) {
+  for (var index = 0, max = quotationList.length, quote; index < max; index++) {
```

Comment:
I [open a follow up](https://github.com/mozilla/serviceworker-cookbook/issues/136) to update all the similar code (this one and the one in the server part of API analytics) to highlight the similarities.